Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Login/LogInViewController.swift

```diff
@@ -91,12 +92,25 @@ class LogInViewController: UIViewController {
                     emailLabel.textColor = .colorText
                     passwordLabel.textColor = .red
                 } else {
-                    alertMessage(title: "You are successfully logged in", userMessage: "Success!")
                     passwordLabel.textColor = .colorText
+                    emailLabel.textColor = .colorText
+                    AuthApiManager.sharedInstance.loginAPI(email: userEmail, password: userPassword) {
+                        success, error in
+                        if(success) {
+                            self.alertMessage(title: "Success!", userMessage: "Succesfully logged in")
+                        } else {
+                            self.alertMessage(title: "Something went wrong!", userMessage: error!.localizedDescription)
+                        }
+                    }
+                    
+                    transitionToTabbar()
```

Comment:
This is not needed here, transition logic should be done outside of this. Think of it like someone is presenting the login screen, that someone needs to present the home-tab bar when the user is loggedin.